REPOSITORY-GUIDELINES.md

```diff
@@ -114,33 +117,43 @@ Oversight Committee.
 Note: prior art here from the
 [Kubernetes community](https://github.com/kubernetes/community/blob/master/github-management/kubernetes-repositories.md).
 
-Working groups should be able to create repositories that meet the following
-bar:
+Working groups are allowed to request repositories that meet the following
+requirements. Actual repo creation will be performed by a SC or TOC member after
+verifying the requirements have been met:
```

Comment:
This section still suggests a chicken and egg problem but I defer to the suggestion in https://github.com/knative/community/pull/149#discussion_r441747631 that we split the mechanics out to another doc. #152 tracks this.

Reply:
Tried rewording this again.